charts/redpanda/templates/rbac.yaml

```diff
@@ -179,6 +179,13 @@ annotations:
   {{- toYaml . | nindent 4 }}
     {{- end }}
 rules:
+  - apiGroups:
+      - apps
+    resources:
+      - statefulsets/status
+    verbs:
+      - patch
+      - update
```

Comment:
Could you mention in PR description why extra quotes requires new RBAC rules?

Reply:
`Also fixes https://github.com/redpanda-data/helm-charts/issues/854`